package.json

```diff
@@ -3,6 +3,7 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@material-ui/core": "^4.11.3",
```

Comment:
Why did we have to use  `@material-ui/core`? Can't we use semantic-ui here so that we can easily plug this in to Identity Server Console. WDYT?

Reply:
Fixed with [db6884a](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/6/commits/db6884ad844ada21c644ca983f566f0fa42f8ad4)